src/Sylius/Component/Review/Factory/ReviewFactoryInterface.php

```diff
@@ -21,17 +22,17 @@
 interface ReviewFactoryInterface extends FactoryInterface
 {
     /**
-     * @param mixed $subjectId
+     * @param ReviewableInterface $subject
      *
      * @return ReviewInterface
      */
-    public function createForSubject($subjectId);
+    public function createForSubject(ReviewableInterface $subject);
```

Comment:
👍  for removing optionality of ``$reviewer`` in the second method - if we're creating review with reviewer we should not allow to create it without reviewer 😄  🐃